mods/cnc/maps/gdi05b/gdi05b.lua

```diff
@@ -0,0 +1,221 @@
+AllToHuntTrigger =
+{
+	Silo1, Proc1, Silo2, Silo3, Silo4, Afld1, Hand1, Nuke1, Nuke2, Nuke3, Fact1
+}
+
+GdiBaseDiscoveryTrigger =
+{
+	CPos.New(41,59), CPos.New(40,59), CPos.New(23,59), CPos.New(22,59),
+	CPos.New(41,58), CPos.New(40,58), CPos.New(23,58), CPos.New(22,58), 
+	CPos.New(41,57), CPos.New(40,57), CPos.New(23,57), CPos.New(22,57), 
+	CPos.New(41,56), CPos.New(40,56), CPos.New(23,56), CPos.New(22,56), 
+	CPos.New(41,55), CPos.New(40,55), CPos.New(23,55), CPos.New(22,55), 
+	CPos.New(41,54), CPos.New(40,54), CPos.New(23,54), CPos.New(22,54), 
+	CPos.New(41,53), CPos.New(40,53), CPos.New(23,53), CPos.New(22,53), 
+	CPos.New(41,52), CPos.New(40,52), CPos.New(23,52), CPos.New(22,52), 
+	CPos.New(41,51), CPos.New(40,51), CPos.New(23,51), CPos.New(22,51), 
+	CPos.New(41,50), CPos.New(40,50), CPos.New(41,49), CPos.New(40,49), 
+	CPos.New(39,59), CPos.New(38,59), CPos.New(25,59), CPos.New(24,59), 
+	CPos.New(39,58), CPos.New(38,58), CPos.New(25,58), CPos.New(24,58), 
+	CPos.New(39,57), CPos.New(38,57), CPos.New(25,57), CPos.New(24,57), 
+	CPos.New(39,56), CPos.New(38,56), CPos.New(25,56), CPos.New(24,56), 
+	CPos.New(39,55), CPos.New(38,55), CPos.New(25,55), CPos.New(24,55), 
+	CPos.New(39,54), CPos.New(38,54), CPos.New(25,54), CPos.New(24,54), 
+	CPos.New(39,53), CPos.New(38,53), CPos.New(25,53), CPos.New(24,53), 
+	CPos.New(39,52), CPos.New(38,52), CPos.New(25,52), CPos.New(24,52), 
+	CPos.New(39,51), CPos.New(38,51), CPos.New(25,51), CPos.New(24,51), 
+	CPos.New(39,50), CPos.New(38,50), CPos.New(39,49), CPos.New(38,49)
```

Comment:
Now that https://github.com/OpenRA/OpenRA/pull/7492 is merged you can rebase and make use of the new discovery triggers.